### ISSUE TITLE
the setOrientation option does not working well with barcode

### ISSUE THREAD

**Issue by scarletsky** [journeyapps/zxing-android-embedded#32]
I try to use the setOrientation for scanning, it work with QR code, but when I test on barcode(ISBN code and product barcode), it isn't work until I rotate the barcode 90 degree. 

I do my test on samsung i9250 and xiaomi HM Note, both of them do not work.
And here is my code:
```java
IntentIntegrator integrator = new IntentIntegrator(MainActivity.this);
integrator.setPrompt("Scan a barcode");
integrator.setOrientation(ActivityInfo.SCREEN_ORIENTATION_SENSOR);
integrator.initiateScan();
```

**Comment by rkistner** [comment]
Confirmed. Will release a fix soon.

**Comment by rkistner** [comment]
Fixed in v2.3.0.

**Comment by scarletsky** [comment]
Thanks for the quick response!
